### PR TITLE
docs: Moved ssh userData example

### PR DESCRIPTION
**Pull request by chrisnegus**
**Description**
There was an example in the Karpenter FAQ of how to use userData to add SSH keys to Kubernetes nodes. To make that information more accessible, the example is being moved in this PR to the AWS/user-data file.

**Does this change impact docs?**
- [X] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

### CODE REVIEW

**Comment by chrisnegus** [comment]
Previews:
https://deploy-preview-2620--karpenter-docs-prod.netlify.app/preview/aws/user-data/
https://deploy-preview-2620--karpenter-docs-prod.netlify.app/preview/faq/#can-i-add-ssh-keys-to-a-provisioner

File: website/content/en/preview/AWS/user-data.md [hunk 1]
```diff
@@ -46,6 +47,37 @@ spec:
...
 
+This example adds SSH keys to allow remote login to the node (replace *my-authorized_keys* with your key file):
+
+```yaml
+apiVersion: karpenter.k8s.aws/v1alpha1
+kind: AWSNodeTemplate
+metadata:
+  name: bottlerocket-example
+spec:
+  amiFamily: Bottlerocket
```
Comment (bwagner5) [inline website/content/en/preview/AWS/user-data.md:58]: This won't work with Bottlerocket, only AL2 and Ubuntu (at least in userdata directly). Bottlerocket would need the admin container and then this userdata nested in the admin container's user-data from the regular bottlerocket toml user-data config file.

It's probably sufficient just to show it for AL2 though and leave out Bottlerocket.

File: website/content/en/preview/AWS/user-data.md [hunk 2]
```diff
@@ -46,6 +47,37 @@ spec:
     karpenter.sh/discovery: my-cluster
 ```
 
+This example adds SSH keys to allow remote login to the node (replace *my-authorized_keys* with your key file):
```
Comment (bwagner5) [inline website/content/en/preview/AWS/user-data.md:50]: can you add the blurb about SSM and ec2-instance-connect here too? 

```
You can use Session Manager (SSM) or EC2 Instance Connect to gain shell access to Karpenter nodes.
See [Node NotReady](https://karpenter.sh/preview/troubleshooting/#node-notready) troubleshooting for an example of starting an SSM session from the command line or [EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-set-up.html) documentation to connect to nodes using SSH.
```

File: website/content/en/preview/faq.md [hunk 3]
```diff
@@ -91,26 +91,8 @@ Karpenter does not offer a way to add SSH keys via provisioners or secrets to th
 However, you can use Session Manager (SSM) or EC2 Instance Connect to gain shell access to Karpenter nodes.
```
Comment (bwagner5) [inline website/content/en/preview/faq.md:91]:
```suggestion
However, you can use Session Manager (SSM) or EC2 Instance Connect to gain shell access to Karpenter nodes.
See [Node NotReady](https://karpenter.sh/preview/troubleshooting/#node-notready) troubleshooting for an example of starting an SSM session from the command line or [EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-set-up.html) documentation to connect to nodes using SSH.
```

File: website/content/en/preview/AWS/user-data.md (hunk 1 above)
Comment (chrisnegus) [reply website/content/en/preview/AWS/user-data.md:58]: Okay. I fixed the metadata.name and amiFamily. Let me know of other parts of that example need to change.

File: website/content/en/preview/AWS/user-data.md (hunk 2 above)
Comment (chrisnegus) [reply website/content/en/preview/AWS/user-data.md:50]: Done.

File: website/content/en/preview/faq.md (hunk 3 above)
Comment (chrisnegus) [reply website/content/en/preview/faq.md:91]: Done

**Comment by ellistarn** [comment]
Needs a rebase, since I renamed this file to `operating-systems.md`. Sorry :(

**Comment by chrisnegus** [comment]
> Needs a rebase, since I renamed this file to `operating-systems.md`. Sorry :(

No problem. I didn't notice. I'll take care of that.

File: website/content/en/preview/AWS/operating-systems.md [hunk 4]
```diff
@@ -46,6 +47,43 @@ spec:
     karpenter.sh/discovery: my-cluster
 ```
 
+This example adds SSH keys to allow remote login to the node (replace *my-authorized_keys* with your key file):
+
+{{% alert title="Note" color="primary" %}}
+Instead of using SSH as set up in this example, you can use Session Manager (SSM) or EC2 Instance Connect to gain shell access to Karpenter nodes.
+See [Node NotReady](https://karpenter.sh/preview/troubleshooting/#node-notready) troubleshooting for an example of starting an SSM session from the command line or [EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-set-up.html) documentation to connect to nodes using SSH.
```
Comment (bwagner5) [inline website/content/en/preview/AWS/operating-systems.md:54]:
```suggestion
See [Node NotReady]({{< ref "../troubleshooting/#node-notready" >}}) troubleshooting for an example of starting an SSM session from the command line or [EC2 Instance Connect](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-connect-set-up.html) documentation to connect to nodes using SSH.
```

File: website/content/en/preview/AWS/operating-systems.md [hunk 5]
```diff
@@ -24,7 +24,8 @@ spec:
 
 **Examples**
 
-Operating systems can be configured using `spec.userData` and `spec.amiSelector` respectively in the `AWSNodeTemplate` resource -
+Your UserData and AMIs can be added to `spec.userData` and `spec.amiSelector` respectively in the `AWSNodeTemplate` resource:
```
Comment (bwagner5) [inline website/content/en/preview/AWS/operating-systems.md:27]:
```suggestion
Operating systems can be configured using `spec.userData` and `spec.amiSelector` respectively in the `AWSNodeTemplate` resource:
```

Comment (chrisnegus) [reply website/content/en/preview/AWS/operating-systems.md:27]: Done.

File: website/content/en/preview/AWS/operating-systems.md (hunk 4 above)
Comment (chrisnegus) [reply website/content/en/preview/AWS/operating-systems.md:54]: Done.

**Comment by chrisnegus** [comment]
> > Needs a rebase, since I renamed this file to `operating-systems.md`. Sorry :(
> 
> No problem. I didn't notice. I'll take care of that.

Done

**Review by bwagner5 — APPROVED** [review]
LGTM

**Pull request merged**